dbms/src/Storages/Page/V3/BlobStore.h

```diff
@@ -126,6 +137,16 @@ class BlobStore : private Allocator<false>
                 type.store(BlobStatType::READ_ONLY);
             }
 
+            bool isBigBlob() const
+            {
+                return type.load() == BlobStatType::BIG_BLOB;
+            }
+
+            void changeToBigBlob()
+            {
+                type.store(BlobStatType::BIG_BLOB);
+            }
```

Comment:
can we set it in the ctor of `BlobStat` when `sm_type==big`?

Reply:
what is `ctor`? i don't get it.

Reply:
`BlobStat::BlobStat(...)`